src/Http/UrlScript.php

```diff
@@ -54,7 +54,8 @@ public function withPath(string $path, string $scriptPath = '')
 	{
 		$dolly = clone $this;
 		$dolly->scriptPath = $scriptPath;
-		return call_user_func([$dolly, 'parent::withPath'], $path);
+		$parent = \Closure::fromCallable([UrlImmutable::class, 'withPath'])->bindTo($dolly);
+		return $parent($path);
```

Comment:
`Closure::call()` exists since PHP 7, so it should be possible to simplify this to

```php
Closure::fromCallable([UrlImmutable::class, 'withPath'])->call($dolly, $path)
```

Reply:
I tried it first. It works only for anonymous functions. 

```
Cannot rebind scope of closure created by ReflectionFunctionAbstract::getClosure()
```

More info: https://bugs.php.net/bug.php?id=74558